### PR TITLE
LRAC 7883 | master

### PR DESCRIPTION
**Pull request by calvinctvo**
https://issues.liferay.com/browse/LRAC-7883

Convert from Cucumber to Poshi based on this test case SyncLiferayDataSourceContacts.feature

### CODE REVIEW

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by calvinctvo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 1]
```diff
@@ -43,6 +29,36 @@ definition {
 		ACUtils.tearDownAC();
 	}
 
+	@description = "Sync Liferay Data Source Contacts"
+	@priority = "5"
+	test CanSyncAllContactsFromDXPToAC {
+		ACUtils.createBlogsAndAddToPage(entryTitle = "Blogs Entry");
```
Comment (joshitagaki) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:35]: Is there a reason to create a blog? Doesn't look like it's used in the test.

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by calvinctvo** [comment]
ci:test:sf

**Pull request closed without merging**